modules/promotion/src/Entity/PromotionInterface.php

```diff
@@ -214,7 +214,18 @@ public function isEnabled();
   public function setEnabled($enabled);
 
   /**
-   * Applies the promotion to an entity.
+   * Checks if the promotion entity can be applied.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The entity.
+   *
+   * @return bool
+   *   Returns TRUE if promotion can be applied, or false if conditions failed.
```

Comment:
We don't need "Returns", that's redundant. Just "TRUE if the promotion can be applied, FALSE otherwise."